packages/elements/src/components/ino-popover/ino-popover.tsx

```diff
@@ -81,26 +94,41 @@ export class Popover implements ComponentInterface {
     this.create();
   }
 
-  // Private methods
+  /**
+   * Emits when an element which is not part of the popover is clicked.
+   * Should be used if you control the state of the popover.
+   */
+  @Event() clickOutside: EventEmitter<void>;
 
   private create() {
-    this.tooltipInstance?.destroy();
+    this.tippyInstance?.destroy();
 
     const target = this.inoFor
       ? document.getElementById(this.inoFor)
       : this.el.parentElement;
 
-    const options = {
+    if (!target && this.inoFor) {
+      console.warn(
+        `The element with the id '${this.inoFor}' could not be found.`
+      );
+    }
+
+    const isControlled = this.inoShow !== undefined;
+
+    const options: Partial<Props> = {
       allowHTML: true,
       appendTo: this.el.parentElement,
       content: this.el,
       duration: 100,
+      hideOnClick: !isControlled,
       placement: this.inoPlacement,
-      trigger: this.inoTrigger,
-      interactive: this.inoInteractive,
+      trigger: isControlled ? 'manual' : this.inoTrigger,
+      interactive: isControlled || this.inoInteractive,
+      showOnCreate: isControlled && this.inoShow,
+      onClickOutside: () => this.clickOutside.emit(),
     };
```

Comment:
For more customization, won't you add a distance property to configure the tippys offset? 

` options['offset'] = [0, this.inoDistance];`.

Default distance is 10px which does not apply to all usecases

Reply:
Nice idea but I would rather do it in a separate issue so the new 'feature' will be visible to the consumer in our changelog.

Reply:
https://github.com/inovex/elements/issues/291